aten/src/ATen/test/basic.cpp

```diff
@@ -232,7 +232,7 @@ static void test(Type & type) {
     for (int64_t i = 0; i < tensor.numel(); ++i) {
       REQUIRE(tensor[i].equal(one * i));
     }
-    for (size_t i = 0; i < tensor.numel(); ++i) {
+    for (size_t i = 0; i < (uint64_t) tensor.numel(); ++i) {
```

Comment:
More idiomatic, and inline with the code below, is `static_cast<uint64_t>(tensor.numel())`